ui.frontend/src/components/CodeArgumentInput.tsx

```diff
@@ -63,6 +64,31 @@ const CodeArgumentInput: React.FC<CodeArgumentInputProps> = ({ arg }) => {
           )}
         />
       );
+    } else if (isDateTimeArgument(arg)) {
+      return (
+        <Controller
+          name={arg.name}
+          control={control}
+          rules={controllerRules(arg)}
+          render={({ field, fieldState }) => (
+            <View key={arg.name} marginBottom="size-200">
+              <DatePicker
+                {...field}
```

Comment:
Since spectrum picker supports min/max then in the backend we could as well, fine